schema/crdb/multiple-default-ip-pools-per-silo/up01.sql

```diff
@@ -0,0 +1,16 @@
+-- Allow multiple default IP pools per silo (one per pool_type + ip_version combo)
+--
+-- Previously, a silo could only have one default IP pool. With this change,
+-- a silo can now have up to 4 default pools: one for each combination
+-- of pool_type (unicast/multicast) and ip_version (v4/v6).
+--
+-- For unicast-only use today, this allows separate default pools for IPv4 and
+-- IPv6, preparing the schema for future multicast support.
+
+-- Drop the old single-default constraint
+DROP INDEX IF EXISTS omicron.public.one_default_ip_pool_per_resource;
```

Comment:
I'm not certain that it matters, but we have generally had only one statement per SQL update file. I'd probably split these and the contents of `up03.sql` into two files.